views/default/js/event_calendar/event_calendar.php

```diff
@@ -148,7 +148,9 @@
 							var msg = res.message;
 							if (success) {
 								var button_text = res.button_text;
+								var button_class = res.button_class;
```

Comment:
The logic that handles the outcome of the change is already implemented in the action. I just added the change of the class to the result. This seemed the easiest way to get it working.